simulation/__main__.py

```diff
@@ -61,6 +61,13 @@ def restrict_between_0_and_1(input_value: float):
         type=str,
     )
 
+    parser.add_argument(
+        "--override-heat-pump-suitability",
```

Comment:
Can we rename this? I think the `help` string still makes sense with this renaming, but the flag is clearer in terms of its effect on the code.
```suggestion
        "--all-agents-heat-pump-suitable",
```